paypal/standard/ipn/views.py

```diff
@@ -31,6 +35,11 @@ def ipn(request):
     flag = None
     ipn_obj = None
 
+    # Avoid the RawPostDataException. See original issue for details:
+    # https://github.com/spookylukey/django-paypal/issues/79
+    assert request.META.get('CONTENT_TYPE', '').startswith(
+        'application/x-www-form-urlencoded'), CONTENT_TYPE_ERROR
+
```

Comment:
I'd say it should either be `AssertionError` or `ValueError`, I don't think `TypeError` is appropriate. I think `AssertionError` is best, we are saying "we are assuming X is true, and can't work correctly otherwise, so are asserting that X is true" (where X is "PayPal will send POST data URL-encoded").